app/app.go

```diff
@@ -137,6 +137,9 @@ import (
 	wasmkeeper "github.com/terra-money/core/x/wasm/keeper"
 	wasmtypes "github.com/terra-money/core/x/wasm/types"
 
+	feesharekeeper "github.com/terra-money/core/x/feeshare/keeper"
```

Comment:
Change "terra-money" namespace to "classic-terra"

Reply:
I don't believe this is possible with the go module still being `module github.com/terra-money/core`. Unless that is what you want me to do here for all other lines in the codebase as well

Reply:
Merge this branch into yours and you should be good to go @ https://github.com/classic-terra/core/tree/correct-module-path :)

Reply:
resolved in ff31c7bc3d3143808915f434139a5e4330caf1ab